### ISSUE TITLE
LLamaIndex + Langflow conflict on SQLAlchemy

### ISSUE THREAD

**Issue by mkamalov** [logspace-ai/langflow#639]
I'm new to Python, so please don't judge me harshly, I'm from the Java world.

Langchain depends on SQLAlchemy 2.0.15, but LLamaIndex depends on 1.4.41

Langflow install log ERROR, if Langflow package is installed after LLamaIndex package:
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behavior is the source of the following dependency conflicts.
llama-index 0.7.5 requires sqlalchemy>=2.0.15, but you have sqlalchemy 1.4.41 which is incompatible.
```

If LLamaIndex installs after Langflow, then Langflow doesn't launch with errors in logs.

In my opinion, LLamaIndex is a valuable package in LLM world, and it is crucial behavior for the Langflow. LangChain and LLamaIndex are strongly integrated into each other and Langflow has to follow the industry and avoid any conflicts between Langflow and LLamaIndex. So I report it as a bug.

Langflow depends on sqlmodel 0.0.8 that depend on SQLAlchemy >=1.4.17,<=1.4.41.

Please, could you fix it?

**Comment by dafinley** [comment]
Have you tried updating pip then trying to install langflow

```
pip3 install --upgrade pip
pip install langflow
```

**Comment by matthias** [comment]
This still exists:

`Because no versions of llama-index match >0.8.40,<0.9.0
 and llama-index (0.8.40) depends on sqlalchemy (>=2.0.15), llama-index (>=0.8.40,<0.9.0) requires sqlalchemy (>=2.0.15).
And because sqlmodel (0.0.8) depends on SQLAlchemy (>=1.4.17,<=1.4.41)
 and no versions of sqlmodel match >0.0.8,<0.0.9, llama-index (>=0.8.40,<0.9.0) is incompatible with sqlmodel (>=0.0.8,<0.0.9).
So, because langflow depends on both sqlmodel (^0.0.8) and llama-index (^0.8.40), version solving failed.`

**Comment by matthias** [comment]
This issue is solved as sqlmodel 0.14 now supports sqlalchemy 2.x
and langflow is using sqlmodel 0.14+ 

```shell
poetry add llama-index
```
